android/tangram/build.gradle

```diff
@@ -14,44 +14,65 @@ version = VERSION_NAME
 apply from: 'versioning.gradle'
 
 android {
-  compileSdkVersion 24
-
-  /*
-   * Ubuntu can't run the aapt on 64 bit before installing this packages
-   *    sudo apt-get install lib32stdc++6
-   *    sudo apt-get install lib32z1
-   */
-
-  buildToolsVersion '24.0.2'
+  compileSdkVersion 25
+  buildToolsVersion '25.0.0'
 
   defaultConfig {
     minSdkVersion 15
-    targetSdkVersion 24
+    targetSdkVersion 25
     versionCode buildVersionCode()
     versionName VERSION_NAME
+    externalNativeBuild {
+      cmake {
+        targets 'tangram'
+        arguments '-DPLATFORM_TARGET=android',
+                  '-DANDROID_TOOLCHAIN=clang',
+                  '-DANDROID_STL=c++_shared'
+        cppFlags '-std=c++14',
+                 '-pedantic',
+                 '-fPIC',
+                 '-fexceptions',
+                 '-frtti'
+        abiFilters 'armeabi-v7a'
+      }
+    }
   }
 
-  sourceSets.main {
-    manifest.srcFile 'AndroidManifest.xml'
-    java.srcDirs = ['src']
-    jni.srcDirs = []
-    jniLibs.srcDirs = ['libs']
+  externalNativeBuild {
+    cmake {
+      path '../../CMakeLists.txt'
+    }
   }
-}
 
-// Add gdb server to apk
-afterEvaluate {
-    Sync packageTask = project.getTasks().findByName("packageReleaseJniLibs")
-    if (packageTask) { packageTask.include(['**/gdbserver', '**/gdb.setup']) }
+  buildTypes {
+    release {
+      externalNativeBuild {
+        cmake.arguments '-DCMAKE_BUILD_TYPE=Release'
+      }
+    }
+    debug {
+      externalNativeBuild {
+        cmake.arguments '-DCMAKE_BUILD_TYPE=Debug'
```

Comment:
Do these take care of debugging for demo app and also debugging on the device, which was disabled here: https://github.com/tangrams/tangram-es/pull/1152/commits/b859576c1f7268b35ce509f0d10c195ea2fdd489